network/dag/bbolt_tree.go

```diff
@@ -115,32 +114,6 @@ func (store *bboltTree) dagObserver(ctx context.Context, transaction Transaction
 	return nil
 }
 
-// migrate builds a tree by walking over the dag and adding all Transaction references to the tree without checking for validity.
-// The tree is stored on disk once it is in sync with the dag.
-func (store *bboltTree) migrate(ctx context.Context, state State) error {
```

Comment:
This should be replaced with something else to allow rebuilding the tree

Reply:
I'll create an issue for that

Reply:
True. Maybe not needed then...